frontend/src/entry/model/settings/ModelRolesInfo.tsx

```diff
@@ -0,0 +1,76 @@
+import { Box, Grid, Typography } from '@mui/material'
+import { Stack } from '@mui/system'
+import { useGetModelRoles } from 'actions/model'
+import { useCallback } from 'react'
+import Loading from 'src/common/Loading'
+import MessageAlert from 'src/MessageAlert'
+import { RoleKind } from 'types/types'
+
+interface ModelRolesInfoProps {
+  modelId: string
+}
+
+export default function ModelRolesInfo({ modelId }: ModelRolesInfoProps) {
+  const { modelRoles, isModelRolesLoading, isModelRolesError } = useGetModelRoles(modelId)
+
+  const roles = useCallback(
+    (kind: string) => {
+      return modelRoles
+        .filter((modelRole) => modelRole.kind === kind)
+        .map((filteredModelRole) => (
+          <Box key={filteredModelRole.id}>
+            <Typography fontWeight='bold'>{filteredModelRole.name}</Typography>
+            <Typography>{filteredModelRole.description}</Typography>
+          </Box>
+        ))
+    },
+    [modelRoles],
+  )
+
+  if (isModelRolesError) {
+    return <MessageAlert message={isModelRolesError.info.message} severity='error' />
+  }
+
+  return (
+    <>
+      {isModelRolesLoading && <Loading />}
+      {!isModelRolesLoading && (
+        <Stack spacing={2}>
+          <Typography>
+            Roles in Bailo are split into two categories; standard and dynamic. Standard roles are generic across
+            different schema and are used for determining model permissions for general purpose model upkeep, whereas
+            dynamic roles are created on a per schema basis and used as part of the review process. The dynamic roles
+            presented below are specified on the schema selected for this model and may not apply to other models using
+            a different schema.
+          </Typography>
+          <Grid container spacing={1}>
+            <Grid item sm={6}>
```

Comment:
This only handles screen sizes of `sm` and above. Change `sm` for `xs` and it'll handle all of them.

Reply:
I did this because I wanted `xs` to be set to `12` so it works on _really_ small screens, I just forgot to add `xs` whoops!!